src/main/cpp/util/InstallDirs.cpp

```diff
@@ -167,5 +170,21 @@ path InstallDirs::GetRootDir()
 	return g_root_dir;
 }
 
+/// Reads the XML file at the given path, relative to some other path.
```

Comment:
Maybe “some other path” → “some anchor path”, to make it instantly obvious which path goes where when looking at the parameter names?